addons/account/static/src/js/tour_manager.js

```diff
@@ -0,0 +1,61 @@
+odoo.define('web_tour.account.TourManager', function (require) {
+'use strict';
+
+var ajax = require('web.ajax');
+var session = require('web.session');
+var TourManager = require('web_tour.TourManager');
+var utils = require('web.utils');
+
+TourManager.include({
+    register: function () {
+        var self = this;
+        var args = Array.prototype.slice.call(arguments);
+        var name = args[0];
+        return this._super.apply(this, arguments).then(function () {
+            var tour = self.tours[name];
+            if (!self.running_tour || !tour || !tour.extra.l10n) {
```

Comment:
What about `extra.fetch_country_code` instead of `extra.l10n`?

Reply:
New way to handle extra data for tours, check BOI advice below